services-camel-app/src/main/webapp/WEB-INF/solr-indexing-context.xml

```diff
@@ -80,6 +80,7 @@
             <list>
                 <ref bean="setRecordDatesFilter" />
                 <ref bean="setDatastreamFilter" />
+                <ref bean="setContentStatusFilter" />
```

Comment:
Adding this is good, but according to the ticket we still need to update SetContentStatusFilter to add a new "THUMBNAIL_ASSIGNED" status. I suspect we should also add a "IS_THUMBNAIL" property to the file itself, just like how IS_PRIMARY_OBJECT is getting set right now, since I think it would make it easier for users to tell what item is the thumbnail in the admin ui.